shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/config/impl/postgresql/ScalingPostgreSQLContainerConfiguration.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.impl.postgresql;
+
+import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+public class ScalingPostgreSQLContainerConfiguration extends DefaultPostgreSQLContainerConfiguration {
+    
+    public ScalingPostgreSQLContainerConfiguration(final String scenario) {
+        super(scenario);
+    }
+    
+    @Override
+    public Map<String, String> getResourceMappings() {
+        return ImmutableMap.<String, String>builder().put("/env/postgresql/postgresql.conf", "/etc/postgresql/postgresql.conf").build();
```

Comment:
Please avoid to use guava API if possible

Reply:
done